prac_06/guitar.py

```diff
@@ -0,0 +1,24 @@
+"""Guitar class"""
+YEAR = 2023
+
+
+class Guitar:
+    """Represent a Guitar class object."""
+
+    def __init__(self, name='', year=0, cost=0):
```

Comment:
similar to previous comment, since you are setting what they are equal to you don't need to set name, year and cost to something. 

Thinking back I am not 100% sure if it is ok by Lindsay but I am pretty sure.